docker/README.md

```diff
@@ -183,3 +183,40 @@ Currently, this the `_FILE` suffix is supported for:
 -   `IOTA_AUTH_PASSWORD`
 -   `IOTA_AUTH_CLIENT_ID`
 -   `IOTA_AUTH_CLIENT_SECRET`
+
+## Best Practices
+
+### Increase ULIMIT in Production Deployments
+ 
+IoTAgent - JSON server normally expects the following changes to ulimits:
```

Comment:
Unclear what `ulimits` actually are 
-  add an introductory sentence describing what they solve
-  add a link to more information e.g. https://docs.docker.com/config/containers/resource_constraints/